.github/workflows/docker-image-build-push.yml

```diff
@@ -0,0 +1,31 @@
+# Build and push docker image
+name: Docker Image CI
+
+on:
+  push:
+    branches:
+      - production
+      - master
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: |
+            ${{ secrets.DOCKER_USERNAME }}/atd-bond-reporting:production
+            ${{ secrets.DOCKER_USERNAME }}/atd-bond-reporting:staging
+            ${{ secrets.DOCKER_USERNAME }}/atd-bond-reporting:latest
+      - name: Image digest
+        run: echo ${{ steps.docker_build.outputs.digest }}
```

Comment:
It's the least important thing, but you might put an end-of-file line feed here to get GitHub to remove its indication that it's missing.